server/sonar-server/src/main/java/org/sonar/server/organization/OrganizationCreationImpl.java

```diff
@@ -220,6 +239,21 @@ private void insertGroupPermission(DbSession dbSession, PermissionTemplateDto te
     dbClient.permissionTemplateDao().insertGroupPermission(dbSession, template.getId(), group == null ? null : group.getId(), permission);
   }
 
+  private void insertQualityProfiles(DbSession dbSession, OrganizationDto organization) {
+    List<ActiveRuleChange> changes = new ArrayList<>();
+    definedQProfileRepository.getQProfilesByLanguage().entrySet()
+      .stream()
+      .flatMap(entry -> entry.getValue().stream())
+      .forEach(profile -> insertQualityProfile(dbSession, profile, organization, changes));
+    activeRuleIndexer.index(changes);
```

Comment:
No sure I agree.

Leaving the indexing to the responsibility of `DefinedQProfileCreation` enforces the indexing to be done one QualityProfile at a time.

When in the current state:
* in `OrganizationCreationImpl`: all QualityProfile of the organization are indexed at once
* in `RegisterQualityProfile`: all QualityProfile of all languages for all organizations at once